docs/modules/ROOT/pages/static-site-generators.adoc

```diff
@@ -0,0 +1,94 @@
+
+Antora supplies Asciidoctor extensions with extensive context information about the file being processed and the set of source files.
+
+AsciiDoc pages transformed to embeddable HTML are wrapped using resources, including templates, from a separately constructed UI bundle.
+Header attributes beginning with `page-` are made available as variables in the template system.
+The UI bundle may contain additional Javascript helper functions that can access variables from the page as well as global context information.
+
+Antora does not support multi-language sites and support is not amenable to implementation through an extension.
+
+Antora does not support blog-like sites natively but support can easily be added through the extension mechanism.
+
+=== xref:jekyll-integration::index.adoc[Jekyll integration]
+
+The Asciidoctor Jekyll integration (Ruby) consists of a Jekyll plugin that uses Asciidoctor to transform source to HTML.
+
+Jekyll supports sources from a single file system tree.
+There is limited location mapping support using a `permalink` page variable.
+
+Layout templates, in the `Liquid` template system, are part of a Jekyll site project.
+Header attributes beginning with `page-` are made available to the template system.
+
+Jekyll does not support multi-language sites in any obvious way.
+
+Jekyll has some native blog support, although this does not affect it's use of Asciidoctor.
+
+=== xref:maven-tools::index.adoc[Maven-tools]
+
+Maven is a very flexible build system and the maven-tools project allows use as a somewhat limited site generator in several ways.
+However, it appears that all uses are page-by-page with no possibility of communicating information about the set of pages.
+
+Using suitable maven plugins it's possible to pull sources in as artifacts from a maven repository.
+
+There appears to be no way to use document header attributes as template variables.
+
+=== https://github.com/middleman/middleman-asciidoc[Middleman integration]
+
+The Asciidoctor Middleman integration (Ruby) consists of a Middleman plugin that uses Asciidoctor to transform source to HTML.
+
+Configuration of Middleman is done through ruby scripting.
+
+Middleman supports sources from a single file system tree.
+It is possible to install an extension providing limited location mapping support using a `permalink` page variable.
+
+Layout templates are part of a Middleman site project.
+Header attributes beginning with `page-` are made available to the template system.
+
+Middleman does not support multi-language sites in any obvious way.
+
+Middleman has some native blog support, although this does not affect it's use of Asciidoctor.
+
+=== Unaffiliated site generators
+
+A list of site generators using Asciidoctor in some capability is maintained https://gist.github.com/briandominick/e5754cc8438dd9503d936ef65fffbb2d[here].
```

Comment:
I'm a little hesitate about linking to this gist, though I don't have a good alternative suggestion. So perhaps it's the best resource we have. Perhaps we can say this is a community-maintained reference.